src/diagnostic-channel-publishers/package.json

```diff
@@ -4,16 +4,15 @@
   "main": ".dist/src/index.js",
   "types": ".dist/src/index.d.ts",
   "scripts": {
-    "build": "tsc --outDir ./.dist && node copyTestAssets.js",
-    "lint": "tslint -c tslint.json -p tsconfig.json",
+    "build": "node node_modules/typescript/bin/tsc --outDir ./.dist && node copyTestAssets.js",
```

Comment:
Looks like it worked fine with `diagnostic-channel/package.json` so I think it should be fine.